ingest-app/src/cmr/ingest/services/bulk_update_service.clj

```diff
@@ -114,13 +115,32 @@
     (when (seq concepts)
       (distinct (map :concept-id concepts)))))
 
+(defn- get-validated-concept-ids
+  "Returns the concept-ids that are valid. Throws exception if there exists invalid
+   concept-ids."
+  [context concept-ids provider-id]
+  (if (= ["ALL"] (map string/upper-case concept-ids)) 
+    (get-provider-concept-ids context provider-id)
+    (let [err-msgs 
+           (for [id concept-ids
+                 :let [err-msg (concepts/concept-id-validation id)
+                       err-msg (if err-msg
+                                 err-msg
+                                 (when-not (string/starts-with? id "C")
+                                   [(str "Collection concept-id " id " does not start with C")]))]
```

Comment:
Perhaps the error message should read something like "Concept id [S1111-PROV1] is not a valid collection concept id, must start with C."

Reply:
Done.